packages/tasks/src/local-apps.ts

```diff
@@ -127,6 +127,39 @@ const snippetLocalAI = (model: ModelData, filepath?: string): LocalAppSnippet[]
 	];
 };
 
+const snippetDiffusionKit = (model: ModelData): LocalAppSnippet[] => {
+	const command = (binary: string) =>
+		[
+			"# Load and run the model:",
+			`${binary} \\`,
+			`  --model-version ${model.id}" \\`,
+			'  --prompt "a futuristic cityscape" \\',
+			`  --height 768 \\`,
+			`  --width 1360 \\`,
+			`  --seed 1001 \\`,
+			`  --step ${model.tags.includes("flux") ? 4 : 50} \\`,
+			`  --cfg ${model.tags.includes("flux") ? 0. : 5} \\`,
+			`  --output ~/Desktop/out.png`,
+		].join("\n");
+	return [
+		{
+			title: "Install from pip",
+			setup: "pip install diffusionkit",
+			content: command("diffusionkit-cli"),
```

Comment:
if binary is always the same mayber simpler to hardcode it a `const command = string`